tests/2d_examples/test_2d_T_shaped_pipe_any_direction/src/T_shaped_pipe_any_direction.cpp

```diff
@@ -97,25 +97,28 @@ SegmentFace inflow_face(inlet_points, rotatePointAroundPoint(Vecd(1, 0), rotate_
 //	Define case dependent body shapes.
 //----------------------------------------------------------------------
 /** Water block body definition. */
-class WaterBlock : public TransformShape<MultiPolygonShape>
+class WaterBlock : public TransformShape<TransformShape<MultiPolygonShape>>
```

Comment:
` TransformShape<TransformShape<MultiPolygonShape>>` is correct ?

Reply:
It is two successive transformations and works.